elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTransactionTest.java

```diff
@@ -222,4 +224,47 @@ public void loadObjectsBypassCache() {
         Mockito.verify(queryLogger, times(1)).completeQuery(
                 Mockito.eq(scope.getRequestId()), any());
     }
+
+    @Test
+    public void loadObjectsExceptionThrownTest() throws Exception {
+        Mockito.reset(queryLogger);
+        Object result = null;
+        try {
+            query = Query.builder().table(playerStatsTable).bypassingCache(true).build();
+            SQLQuery myQuery = SQLQuery.builder().clientQuery(query)
+                    .fromClause(query.getTable().getName())
+                    .projectionClause(" ").build();
+
+            when(queryEngine.executeQuery(query, qeTransaction)).thenReturn(null);
+            when(queryEngine.explain(query)).thenReturn(myQuery.toString());
+            AggregationDataStoreTransaction transaction =
+                    new MyAggregationDataStoreTransaction(queryEngine, cache, queryLogger);
+            EntityProjection entityProjection = EntityProjection.builder().type(PlayerStats.class).build();
+
+            transaction.loadObjects(entityProjection, scope);
+
+        } catch (Exception e) {
+                result = e.getMessage();
+        } finally {
+            Object finalResult = result;
```

Comment:
I don't think we need this variable.

Reply:
The local variable that we are referencing from inside the lambda expression should be final. That is why we have this. Otherwise, we can also check for Mockito.verify() from inside the catch block. We will not need to have the `finalResult` local variable then.